source/getting-started/tutorials/timing-manager/index.md

```diff
@@ -0,0 +1,381 @@
+
+To link the ADC to the [Timing Manager](/firmware/arch/timing-manager.md), edit the  `...controller_init()` function to include the function call `timing_manager_enable_sensor(ADC)`:
+
+`app_controller.c`:
+```C
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+
+### Reading Sensor Data
+Data can be obtained from the sensor interaces in the usual manner, irrespective of the mode of the [Timing Manager](/firmware/arch/timing-manager.md). For example, the ADC can be read via `analog_getf(ANALOG_IN1, &output)` (see the [analog input page](/hardware/subsystems/analog.md)). Note that we won't be using the ADC's actual numerical output in this tutorial, we're just enabling it to activate the [Timing Manager](/firmware/arch/timing-manager.md).
+
+### Reporting sensor acquisition time and sensor data staleness
+
+We're also going to add functionality to report how old the ADC data is as well as how long it took the sensor to acquire that data.
+
+`task_controller.h` at the top:
+```C
+extern uint8_t sensorFlag;
+```
+
+`task_controller.c` in `task_controller_callback(void *arg)`:
+```C
+if (sensorFlag) {
+    cmd_resp_printf("ADC time to acquire: %lfus\n", timing_manager_get_time_per_sensor(ADC));
+    cmd_resp_printf("ADC time since done: %lfus\n", timing_manager_get_time_since_sensor_poll(ADC));
+    sensorFlag = 0;
+}
+```
+
+`cmd_controller.c` at the top:
+```C
+static uint8_t ctrl_initialized = 0;
+```
+
+`cmd_controller.c` in `int cmd_vsiApp(int argc, char **argv)`:
+```C
+if (argc == 3 && strcmp("sensor", argv[1]) == 0 && strcmp("timing", argv[2]) == 0) {
+    if (ctrl_initialized == 0) {
+        cmd_resp_printf("ctrl must be initialized\n");
+        return CMD_FAILURE;
+    }
+    sensorFlag = 1;
+    return CMD_SUCCESS_QUIET;
+}
+```
+
+`cmd_controller.c` in `int cmd_vsiApp(int argc, char **argv)` (update existing init and deinit code):
+```C
+if (argc == 2 && strcmp("init", argv[1]) == 0) {
+    if (task_controller_init() != SUCCESS) {
+        return CMD_FAILURE;
+    }
+    ctrl_initialized = 1;
+    return CMD_SUCCESS;
+}
+
+if (argc == 2 && strcmp("deinit", argv[1]) == 0) {
+    if (task_controller_deinit() != SUCCESS) {
+        return CMD_FAILURE;
+    }
+    ctrl_initialized = 0;
+    return CMD_SUCCESS;
+}
+```
+
+We can then use the command `ctrl sensor timing` to get the information.
+
+```
+ADC time to acquire: 0.820000us
+ADC time since done: 20.805000us
+```
+
+The ADC time to acquire refers to how long between the trigger for the ADC to start sampling to when it was complete.\
+The ADC time since done is the "staleness" of the data, it refers to how long ago the ADC reported done. This may be useful when controlling a motor with careful timing considerations.
```

Comment:
Two things:

1. Write this as a deliberate step that a user should take (i.e., program your code onto the AMDC and measure the sensor acquisition timing using the newly created `ctrl sensor timing` command). 
2. Move it down into the `Sensor Timing` subsection. I suggest that you start the `Sensor Timing` subsection by telling the user to do two things: 
    1. Run `ctrl sensor timing` to obtain sensor acquisition timing information  
    2. Run `ctrl stats print` to obtain run time and loop time